### PR TITLE
LPS-165580 Made Collection display show all object system field elements

### PR DESCRIPTION
**Pull request by ericthmoritsuka**
Related Issue: https://issues.liferay.com/browse/LPS-165580

### CODE REVIEW

**Comment by ericthmoritsuka** [comment]
ci:test:sf

**Comment by ericthmoritsuka** [comment]
ci:test:sf

**Comment by ericthmoritsuka** [comment]
ci:test:relevant

**Comment by ericthmoritsuka** [comment]
Hey @seltong, I'll take a look at it. I think I know what went wrong, but I do not know how to fix it. Do you have any suggestions? I think I did an amend somewhere I should not have. I guess that is what went wrong.

**Comment by ericthmoritsuka** [comment]
@seltong fixed it. thx

**Comment by ericthmoritsuka** [comment]
ci:test:sf

**Comment by seltong** [comment]
@ericthmoritsuka you can try these steps:

- Go to your branch LPS-165580;
- Type: `git reset HEAD~1`
- Type: `git status`
- Type: `g add ...` -> Add only your changes to stage
- Type: `git checkout .`
- Now you can commit your changes. `git commit -m "LPS message"`
- Type: `git pull -r upstream master`

I believe now your commit is back

**Comment by ericthmoritsuka** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java [hunk 1]
```diff
@@ -159,6 +160,18 @@ private List<InfoFieldValue<Object>> _getInfoFieldValues(
 			new InfoFieldValue<>(
 				ObjectEntryInfoItemFields.userNameInfoField,
 				objectEntry.getUserName()));
+		objectEntryFieldValues.add(
```
Comment (seltong) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java:163]: Hey @ericthmoritsuka, could you please sort this by `ObjectEntryInfoItemFields` constants?

Feel free to message me if you have any questions :+1:

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFormProvider.java [hunk 2]
```diff
@@ -270,6 +270,12 @@ private InfoFieldSet _getBasicInformationInfoFieldSet() {
 			ObjectEntryInfoItemFields.userNameInfoField
 		).infoFieldSetEntry(
 			ObjectEntryInfoItemFields.userProfileImageInfoField
+		).infoFieldSetEntry(
```
Comment (seltong) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFormProvider.java:273]: Hey @ericthmoritsuka, could you please sort this?

**Review by seltong — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java (hunk 1 above)
Comment (ericthmoritsuka) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java:159]: done! thx @seltong

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFormProvider.java (hunk 2 above)
Comment (ericthmoritsuka) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFormProvider.java]: and done. thx @seltong

**Comment by ericthmoritsuka** [comment]
ci:test:relevant

**Comment by seltong** [comment]
Hi @ericthmoritsuka, I will take a look at this pull request as soon as possible.

**Comment by seltong** [comment]
Resent here: https://github.com/liferay-objects/liferay-portal/pull/1281

**Pull request closed without merging**